tests/spectest/spec_test.go

```diff
@@ -20,6 +20,9 @@ import (
 	"github.com/tetratelabs/wazero/wasm/jit"
 )
 
+//go:embed testdata/*
```

Comment:
nit does this accept *.wasm? it will help me in inventory of which tests still use binaries

Reply:
not really, we have to read *.json first and inside of it wasm binary names are specified.

Reply:
https://github.com/tetratelabs/wazero/pull/197/commits/58cdd090272be91978c2fef72ceaf40d10c42172 specified at least!